core/database/src/main/java/social/firefly/core/database/SocialDatabase.kt

```diff
@@ -79,6 +85,9 @@ import social.firefly.core.database.model.entities.statusCollections.SearchedSta
         MainNotification::class,
         MentionListNotification::class,
         FollowListNotification::class,
+        TrendingLink::class,
+        TrendingStatus::class,
+        TrendingHashTag::class,
     ],
     version = 1,
```

Comment:
We need to either increase the version number here, undo the changes to the `1.json` schema, and generate a `2.json` schema

or

Enable destructive migrations in room and make sure it doesn't crash if we don't update the schema version

Reply:
I added destructive migrations but didn't increment the db.  I thought we were going to just keep it at v1 and require clearing data until an official release?

Reply:
I'm fine with destructive migrations until we release, but it would be nice to increment the version so when my app auto-updates through obtainium, it doesn't just crash when I open it.

Reply:
We can roll back to version 1 before a release release